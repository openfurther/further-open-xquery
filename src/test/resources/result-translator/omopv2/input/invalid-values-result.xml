<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ResultList>
	<Person xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
		<personId>1</personId>
		<genderConceptId>1</genderConceptId>
		<locationConceptId>1</locationConceptId>
		<raceConceptId>1</raceConceptId>
		<sourceGenderCode>Male</sourceGenderCode>
		<sourceLocationCode>Moon</sourceLocationCode>
		<sourcePersonKey>key</sourcePersonKey>
		<sourceRaceCode>Blue</sourceRaceCode>
		<yearOfBirth>2013</yearOfBirth>
	</Person>
</ResultList>
