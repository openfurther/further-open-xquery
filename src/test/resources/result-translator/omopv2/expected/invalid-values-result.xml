<error xmlns="http://further.utah.edu/core/ws">
 <code>DTS_RESULT_TRANSLATION_ERROR</code>
 <message>DTS Mapping for [ OMOP-V2.administrativeGender ] May be Missing</message>
</error>